oss-vienna-happy-hour.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: 'Postgres Happy Hour with Xata at Open Source Summit Europe'
+description: 'Join us for the Postgres Happy Hour in Vienna on September 18th!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
+  alt: Postgres Happy Hour with Xata at Open Source Summit Europe
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 09-03-2024
+tags: ['postgres', 'oss', 'vienna', 'happy-hour', 'xata', 'postgresql', 'postgres-happy-hour', 'ossumit', 'booth']
+published: true
+slug: oss-vienna-happy-hour
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
+---
+
+## Postgres Happy Hour with Xata at Open Source Summit Europe
+
+On September 16-18th, we'll be at the [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria.
+
+We will have a booth in the showcase area on the ground floor where we'd love to chat with you about Postgres and Xata.
+Personally, this event is extra special for me, as it will be my first time doing booth duty as a new Xata employee.
+If you're attending the Open Source Summit, please stop by our booth and say hello!
+
+We are open source enthusiasts here at Xata, and here are a few of our projects:
+
+- [pgroll](https://github.com/xataio/pgroll): An open source PostgreSQL tool for zero-downtime, reversible schema migrations.
+- [pgstream](https://github.com/xataio/pgstream): An open source change data capture (CDC) command-line tool and library for Postgres.
+- [pgzx](https://github.com/xataio/pgzx): An open source framework to easily build PostgreSQL extensions in Zig, a systems programming language for maintainable and fast code.
+
+If you'd like to set up a time to chat about anything related to Postgres, Xata, or open source in general, feel free to email me at gulcin@xata.io.
+
+## Join us for the Postgres Happy Hour!
+
+It is not only booth duty, but I am happy to announce that we'll be also hosting a [Postgres Happy Hour at the Open Source Summit](https://events.linuxfoundation.org/open-source-summit-europe/features/co-located-events/#postgres-happy-hour-with-xata) on September 18th!
+
+At Xata, we are passionate about PostgreSQL and its role in powering modern applications.
```

Comment:
I feel like this entire paragraph feels a little too chat GPT like... It's saying a lot without really saying much and is doing so with many common buzzwords. Instead it would be nice to make it more personal as to the reasons why Xata is hosting this happy hour at this event... what are our specific goals / aims?

Reply:
I don't know how to write this better. We are hosting the event to celebrate Postgres project & community and get to meet people who might be interested in Postgres and Xata. The goal is to explain what we do at Xata to the people who comes. 

How would you write this paragraph? Open to suggestions.